File: problem001/solution.c
#include <stdio.h>

int main(void)
{
    int x, sum;
    
    for(x = 1, sum = 0; x < 1000; x++){
        if(x % 3 == 0 || x % 5 == 0) {
            sum += x;
        }
    }

    printf("Solution: %d\n", sum);
    return 0;
}

File: problem001/solution.php
<?php

require "../src/helpers.php";

for($x = 1, $sum = 0; $x < 1000; $x++){
	if($x % 3 == 0 || $x % 5 == 0) {
		$sum += $x;
	}
}
solution($sum);
